src/handlebars/getinvolved.handlebars

```diff
@@ -7,31 +7,32 @@
   <div class="margin-bottom">
     <h2 class="bold">I'm a Java / OpenJDK User</h2>
     <p>
-      We provide solidly made binaries to the highest quality possible utilising existing OpenJDK test suites
-      (via <em>jtreg</em> tests, <em>jcov</em> test coverage et al) and other OSS donated test suites. However, these binaries are
-      <strong>not</strong> commercially supported.  See <a href="./support.html">Support</a> for details.
+      We provide rock-solid binaries to the highest quality possible utilising existing OpenJDK test suites
+      (via <em>jtreg</em> tests, <em>jcov</em> test coverage et al) and over 60,000 other OSS donated test suites.
```

Comment:
I know you didn't change it, but consider changing "over 60,000 other OSS donated test suites" to ""over 60,000 additional test suites".

They are OSS, but that acronym may not be universally recognised.